SPEC.md

```diff
@@ -128,6 +129,7 @@ The network configuration is described in JSON form. The configuration can be st
 - `cniVersion` (string): [Semantic Version 2.0](http://semver.org) of CNI specification to which this configuration conforms.
 - `name` (string): Network name. This should be unique across all containers on the host (or other administrative domain).
 - `type` (string): Refers to the filename of the CNI plugin executable.
+- `args` (dictionary): Optional additional arguments provided by the container runtime. For example a dictionary of labels could be passed to CNI plugins by adding them to a labels field under `args`.
```

Comment:
I would say that this is completely up to the plugin developers and should go in the respective documentation. I don't have an opinion if we should suggest or even dictate a precedence order, FWIW the spec already says on a different line `args` is to be preferred over `CNI_ARGS`